experimental/datacommons-js/example.html

```diff
@@ -22,7 +22,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.21/lodash.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.0/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/4.6.2/js/bootstrap.min.js"></script>
-    <script src="https://autopush.datacommons.org/datacommons.js"></script>
+    <script src="https://datacommons.org/datacommons.js"></script>
```

Comment:
Reviewers: Do we want to make this change here? Or should we stick to autopush?

Reply:
Let's use `datacommons.org` since it will be deployed early next week